File: sandbox/python/src/TemplateFilesDefaultExample.py
from datetime import date, datetime
from pprint import pprint

from dropbox_sign import ApiClient, ApiException, Configuration, api, models

configuration = Configuration()

with ApiClient(configuration) as api_client:
    try:
        response = api.TemplateApi(api_client).template_files(
            template_id="f57db65d3f933b5316d398057a36176831451a35",
            file_type=None,
        )

        open("file_response.zip", "wb").write(response.read())
    except ApiException as e:
        print("Exception when calling Template#template_files: %s\n" % e)

File: sandbox/node/src/TemplateFilesDefaultExample.ts
import * as fs from 'fs';
import api from "@dropbox/sign"
import models from "@dropbox/sign"

const apiCaller = new api.TemplateApi();

const templateId = "f57db65d3f933b5316d398057a36176831451a35";
const fileType = undefined;

apiCaller.templateFiles(
    templateId,
    fileType,
).then(response => {
  fs.createWriteStream('file_response.zip').write(response.body);
}).catch(error => {
  console.log("Exception when calling Template#templateFiles:");
  console.log(error.body);
});
